schedule/yast/raid/raid1_opensuse_lvm.yaml

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
name of the file: `raid1_lvm_opensuse.yaml. In fact it should be `raid1+lvm_opensuse` to respect the original name of the test usite, but we can accept the underscore too. Alex's nomenclature I think is like that, the name of the original testsuite plus _product_ plus _something_of_that_test_suite.

Reply:
done

Reply:
Please, before saying that is "done" and close the conversation, it would be better if you push the change in order to review it properly.